.github/workflows/ubuntu-2004.yml

```diff
@@ -15,6 +15,14 @@ jobs:
     name: Ubuntu 20.04 | Build
     runs-on: ubuntu-latest
     steps:
+      - name: Check for ccache disable
+        id: no-ccache
+        uses: saulmaldonado/skip-workflow@v1
```

Comment:
Since this gives wide write access to the repo, and it isn't from a highly trusted source (like github or docker), it'd probably be prudent to lock this down to a specific revision.